pmoves/services/hi-rag-gateway-v2/routes/geometry.py

```
@@ -0,0 +1,574 @@
+        try:
+            await ws.close()
+        except Exception:
+            pass
+        try:
+            await _room_broadcast_roster(room)
+        except Exception:
+            pass
+
+
+@router.post("/mesh/handshake")
+def mesh_handshake(body: Dict[str, Any], _=Depends(require_admin_tailscale)):
```

Ruff (0.15.10)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)